tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -4,6 +4,73 @@
 import argparse
 import sys
 import os
+import re
+from pathlib import Path
+
+
+# What does the names of our C API tarball/zip files looks like
+# os: win, linux, osx
```

Comment:
shouldn't our tarball and zip packages have -gpu suffix for name ? (not cuda/tensorrt which are internal intermediate packages?)

Reply:
Yes, this function consumes the internal intermediate packages and produce a nuget package from that. This comment was written before the TRT work.